### ISSUE TITLE
Carousel.Slide `x.clickAllowed` is not a function

### ISSUE THREAD

**Issue by SomiDivian** [mantinedev/mantine#4174]
### What package has an issue

@mantine/carousel

### Describe the bug

"@mantine/carousel": "^6.0.9",
"embla-carousel-react": "^8.0.0-rc01"

Error
```cli
CarouselSlide.tsx:51 Uncaught TypeError: _a2.clickAllowed is not a function
```

### What version of @mantine/hooks page do you have in package.json?

6.0.9

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rtivital** [comment]
use embla-carousel-react 7.x instead of 8.x

**Comment by davidjerleke** [comment]
Hi!

Just adding my two cents. The [release notes](https://github.com/davidjerleke/embla-carousel/releases/tag/v8.0.0-rc01) for v8 state that `clickAllowed` has been removed and is now handled automatically. I don’t think downgrading Embla is a future proof solution but rather band aid for now.

**Comment by rtivital** [comment]
Yes, it will be fixed in `@mantine/carousel` once embla 8.0.0 is out, for now, it is required to use 7.0

**Comment by davidjerleke** [comment]
I understand. Thanks for the prompt reply 👍🏻.

Cheers

**Comment by yunusirmak** [comment]
You can also avoid this issue by giving 

```javascriptreact
        onClick={(e) => {
          e.stopPropagation();
        }}
```
to your carousel slide items. This fixed the issue without downgrading the package.

Here the full code for reference:
```javascriptreact
  const slides = images.map((url) => (
    <Carousel.Slide key={url}>
      <Image
        onClick={(e) => {
          e.stopPropagation();
        }}
        src={url}
        width="100%"
        height={carouselHeight}
        style={{ objectFit: "cover" }}
      />
    </Carousel.Slide>
  ));
```

**Comment by ejixxer999** [comment]
how do I got down to v7 ?

**Comment by rtivital** [comment]
Fixed in 6.0.17 – clickAllowed is no longer called
